superbench/analyzer/data_diagnosis.py

```diff
@@ -311,10 +309,12 @@ def output_diagnosis_in_jsonl(self, data_not_accept_df, output_path):
             data_not_accept_json = data_not_accept_df.to_json(orient='index')
             data_not_accept = json.loads(data_not_accept_json)
             if not isinstance(data_not_accept_df, pd.DataFrame):
-                logger.warning('DataDiagnosis: output json data - data_not_accept_df is not DataFrame.')
-                return
+                logger.log_and_raise(
+                    Exception, msg='DataDiagnosis: output json data - data_not_accept_df is not DataFrame.'
+                )
             if data_not_accept_df.empty:
-                logger.warning('DataDiagnosis: output json data - data_not_accept_df is empty.')
+                with p.open('w') as f:
+                    pass
```

Comment:
what's purpose of this, create the file?

Reply:
yes, create a empty file if no defective node, in case there're automatic pipeline can not find the file and think it crashed

Reply:
then why not use touch directly

Reply:
because I want to overwrite if the file exists